src/components/Box/Box.Overview.stories.mdx

```diff
@@ -648,6 +650,23 @@ Set the `fontSize` prop with a [font size](/?path=/story/design-tokens-font-size
   </Story>
 </Canvas>
 
+## Font Family
+
+Set the `fontFamily` prop with a [font family](/?path=/story/design-tokens-font-family--page) token to set the size of the text in the Box.
+
+<Canvas>
+  <Story name="Font Family">
+    {[...FONT_FAMILY_OPTIONS].map((fontFamily, i) => (
+      <Box fontFamily={fontFamily} key={i} margin='0 0 md 0' fontSize="xl">
+        {`${fontFamily}`}
+        <Box fontSize="sm">
+          {FONT_FAMILY_VALUES[fontFamily].value}
+        </Box>
+      </Box>
+    ))}
+  </Story>
```

Comment:
I think this story should be added to the "Visual Regression Tests" stories folder of Box

For the docs, I think we should show the exact markup so that it can be more easily copy/pasted.